src/components/demo/DOITooltipDemo.tsx

```diff
@@ -37,20 +37,68 @@ const DOITooltipDemo = (props: Props) => {
         }
     }
 
+    /* Function to map Handle or DOI response to valid record */
+    const FormatResponse = (responseRecord: {[name: string]: any}) => {
+        const record = {
+            data: {
+                attributes: {
+                    referentName: responseRecord.values[16].data.value,
+                    specimenHost: responseRecord.values[18].data.value,
+                    specimenHostName: responseRecord.values[19].data.value,
+                    primarySpecimenObjectId: responseRecord.values[20].data.value,
+                    topicDiscipline: responseRecord.values[27].data.value,
+                    livingOrPreserved: responseRecord.values[29].data.value
+                }
+            }
+        };
+
+        return record;
+    }
+
     /* Function for fetching DOI details */
     const TriggerTooltip = async () => {
         try {
-            const response = await fetch(`https://dev.dissco.tech/handle-manager/api/v1/pids/${doi.replace(process.env.REACT_APP_DOI_URL as string, '')}`);
-            const record = await response.json();
+            /* If DOI does not contain local prefix, try to resolve with the general handler */
+            if (doi.includes('TEST') || doi.includes('SANDBOX')) {
+                let environment: string = 'dev';
 
-            if (record.data) {
-                setActive(true);
+                if (doi.includes('SANDBOX')) {
+                    environment = 'sandbox';
+                }
+
+                const response = await fetch(`https://${environment}.dissco.tech/handle-manager/api/v1/pids/${doi.replace(process.env.REACT_APP_DOI_URL as string, '')}`);
+                const record = await response.json();
+
+                if (record.data) {
+                    setRecord(record);
+                }
+            } else if (doi.includes('20.5000.1025')) {
```

Comment:
if the doi and handle apis have the same structure, maybe just use the handle api for 20.5000.1025 and 10.3535 PIDs?

Reply:
Was thinking about that, left it open because thought maybe there is going to be stuff outside of our reach (don't think so)

Reply:
both the doi api and the handle api are managed externally. we only manage the dissco ones. 

Keep it like this if you want! just wanted to clarify